docs/mine-hnt/validators/wallet.mdx

```diff
@@ -0,0 +1,88 @@
+sidebar_label: Configure wallet
+slug: /mine-hnt/validators/validator-wallet
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Configure wallet
+
+You’ll need an owner wallet to:
+* Stake your test tokens (TNT)
+* Receive validator node consensus group rewards
+
+Note: you can have a single wallet for multiple validator nodes. However multiple wallets for a single validator node is not supported.
+
+For instructions to build the wallet go [here](https://github.com/helium/helium-wallet-rs).
+
+At any time use ```-h``` or ```--help``` to get more help for a command.
+
+## Wallet commands
+
+### Create a wallet
+
+```./bin/helium-wallet create basic wallet --network testnet``` 
+
+### View wallet balance and public key
+```./bin/helium-wallet balance info``` view public key and TNT balance. You’ll need this public key address to obtain TNT needed for staking. 
+
+To obtain TNT tokens visit [Obtain testing token (TNT)](/mine-hnt/validators/validator-tnt).
+
+### Stake tokens
+After you have received your TNT you need to stake these tokens. 
+
+As part of the staking process the validator node address needs to be in the staking transaction and also needs to sign the transaction. After a wallet stakes a validator node, the wallet becomes that node’s owner, has control over that validator node, and receives the validator rewards.
+
+#### To stake tokens perform the following:
+
+* Obtain the validator node address:
+    * On the validator, run ```miner peer addr```.  This will produce something like: ```/p2p/112sSfUCQrVNAfFKVZWUSCeLs67wH8o1xz4yj4nKSPJKBT8bUkXi```
+
+    The part following the final slash, in this example ```112sSfUCQrVNAfFKVZWUSCeLs67wH8o1xz4yj4nKSPJKBT8bUkXi```, is the validator node's public address.
+
+* Stake the TNT:
+    * ```./bin/helium-wallet validators stake <public-address> <stake> --commit```
+
+### Unstake tokens
+If you unstake tokens the validator node enters a cooldown period. During this cooldown period the validator node will not earn rewards and at the end of the cooldown period (currently 250,000 blocks or approximately 5 months) your initial staking amount will be returned to the wallet. 
+
+#### To unstake tokens from a validator node perform the following:
+```./bin/helium-wallet unstake <public-address>```
+
+### Validator transfers 
+A wallet can be paired with multiple validator nodes. Validator nodes can be transferred to different wallets. Staking amount can also be transferred to someone else.
+
+#### To transfer a validator's stake to a new validator, or transfer the stake to a validator in another wallet:
+```./bin/helium-wallet validators transfer create --new-address <new-public-address> --old-address <old-public-address>```
+
+To transfer the stake to someone else:
+
+```./bin/helium-wallet --format json validators transfer --new-address <new-public-address> --old-address <old-public-address> --new-owner <new-owner-public-address>```
+
+This will produce an output similar to the following:
```

Comment:
hey @abhay can you provide an example of actual output? Evan provided this dummy data, but as you can see the new and old addresses identical.